features/step_definitions/login_steps.rb

```diff
@@ -1,17 +1,20 @@
 Given(/^i am on the sign in page$/) do
-  
+  @system.loginSlack.visit
+  expect(@browser.find_element(id: "domain").displayed?).not_to eq nil
 end
 
 When(/^i enter the team name$/) do
-  pending # Write code here that turns the phrase above into concrete actions
+  @system.loginSlack.sendTeamName("slack-web-automation2")
+  expect(@browser.find_element(id: "email").displayed?).not_to eq nil
 end
 
 When(/^i enter my valid email and password$/) do
-  pending # Write code here that turns the phrase above into concrete actions
+  @system.loginSlack.sendLoginCredentials('slacktestgerrard@gmail.com','slackpass')
```

Comment:
Can we try to use snake case? Also, it might be sensible to rename the `loginSlack` method to just `login`. cc @cjdpearce